packages/@headlessui-tailwindcss/package.json

```diff
@@ -4,13 +4,16 @@
   "description": "A complementary Tailwind CSS plugin",
   "main": "dist/index.cjs",
   "types": "dist/index.d.ts",
+  "module": "dist/headlessui.esm.js",
   "license": "MIT",
   "files": [
     "README.md",
     "dist"
   ],
   "exports": {
-    "require": "./dist/index.cjs"
+    "import": "./dist/headlessui.esm.js",
+    "require": "./dist/index.cjs",
+    "types": "./dist/index.d.ts"
```

Comment:
The `types` key needs to be listed _first_ to resolve correctly because the package is not `type: "module"`

Reply:
@thecrypticace wow, didn't know that. Changed it!